crates/node/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "restate-node"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+publish = false
+
+[features]
+default = []
+options_schema = ["dep:schemars", "restate-admin/options_schema", "restate-meta/options_schema", "restate-node-ctrl/options_schema", "restate-worker/options_schema", "restate-cluster-controller/options_schema"]
+
+[dependencies]
+
```

Comment:
```suggestion
```

Reply:
👍 Have cleaned it up in a follow-up commit.